src/common/darktable.h

```diff
@@ -245,29 +245,30 @@ typedef float dt_boundingbox_t[4];  //(x,y) of upperleft, then (x,y) of lowerrig
 typedef enum dt_debug_thread_t
 {
   // powers of two, masking
-  DT_DEBUG_CACHE          = 1 <<  0,
```

Comment:
can we keep the clean formatting here ?

Reply:
oops... sorry for that ! should be fixed now...